### PR TITLE
OPSEXP-1742 Document quickstart for detect-secrets tool

### PR DESCRIPTION
**Pull request by gionn**
OPSEXP-1742

### CODE REVIEW

File: docs/security.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+For each detected secret, it will ask you if that secret is really meant to be
+present in the codebase or not:
+
+* Replying with Yes, will whitelist that entry as a non-secret (`is_secret`
+  field will be `false`)
+* Replying with No, will mark that entry as a secret that is meant to be removed
+  in the future (`is_secret` field will be `true`)
+* Replying with Skip, will skip to the next secret without marking the current
+  secret
+
```
Comment (alxgomz) [inline docs/security.md:43]: Can you add here a link to the secret skip section so people can choose to have a dummy password for their repo?
And also linking another section to exclude files based on path (with appropriate warning).

**Review by alxgomz — COMMENTED** [review]
small improvements

File: docs/security.md (hunk 1 above)
Comment (gionn) [reply docs/security.md:49]: not sure to have got everything you asked for, but I've added a link to the exclusion of secrets here and tried to clarify which is the purpose of the skip action while running the auditing command

**Review by alxgomz — APPROVED** [review]
LGTM
@DenysVuika does that PR help or are we still missing some of your concerns?

**Review by arditdomi — APPROVED** [review]

**Pull request merged**